README.md

```diff
@@ -1,91 +1,25 @@
 [![Marketplace Version](https://vsmarketplacebadge.apphb.com/version/ivangsa.asyncapi-preview.svg 'Current Release')](https://marketplace.visualstudio.com/items?itemName=ivangsa.asyncapi-preview) [![Marketplace Downloads](https://vsmarketplacebadge.apphb.com/downloads-short/ivangsa.asyncapi-preview.svg 'Current Release')](https://marketplace.visualstudio.com/items?itemName=ivangsa.asyncapi-preview.svg)
 
-# AsyncAPI Playground Viewer - v0.0.1
+# AsyncAPI Preview
 
-**AsyncAPI Playground Viewer lets you preview AsyncAPI 2.0.0 files as you type in Visual Studio Code. Additionally provide intellisense/linting for the files as well.**
+Preview AsyncAPI documents inside VSCode.
 
-It works on asyncapi files in yaml format. Preview happens in real time as you type.
+AsyncAPI Preview was simplified and reworked from scratch to use the latest [@asyncapi/asyncapi-react](https://github.com/asyncapi/asyncapi-react/tree/next), removing old dependencies on Express, socket.io and js-yaml with better startup preformace and bundle size.
 
-## Requirements
-AsyncAPI Playground Viewer uses https://github.com/asyncapi/generator which requires this software to be installed:
+You can open AsyncAPI Preview from editor title/context menu.
 
-- v12.16+ < Node.js
-- v6.13.7+ < npm
+![AsyncAPI Preview](resources/asyncapi-editor-title-context.png)
```

Comment:
this image is not visible in readme of the VSCode, I wonder if not because of 👇🏼 

```suggestion
![AsyncAPI Preview](./resources/asyncapi-editor-title-context.png)
```

Reply:
The reason it doesn't show in vscode markteplace it's because the image doesn't exist (yet) in the configured git repo (which is this one, not mine)
That should be fixed after merging... but I'm renaming it to an image that already exists so it shows now as well

Reply:
oh, makes sense 🤦🏼 sorry